src/Components/Helper/ComponentHelper.php

```diff
@@ -14,7 +14,7 @@ class ComponentHelper
 
     const ACTION_ID_PATTERN = "/^action\.[A-z]*\.[A-z_]*$/";
     const INTERPRETER_ID_PATTERN = "/^interpreter\.[A-z]*\.[A-z_]*$/";
-    const LANGUAGE_SERVICE_ID_PATTERN = "/^language_service\.core\.[A-z_]*$/";
+    const LANGUAGE_PROCESSOR_ID_PATTERN = "/^language_processor\.core\.[A-z_]*$/";
```

Comment:
These component ID's can be registered with any scope, not just `core`. Let's make this consistent with the established patterns:

```suggestion
    const LANGUAGE_PROCESSOR_ID_PATTERN = "/^language_processor\.[A-z]*\.[A-z_]*$/";
```